api/app/src/main/kotlin/packit/config/RunnerConfig.kt

```diff
@@ -12,5 +12,6 @@ data class RunnerRepository(
 data class RunnerConfig(
     val url: String,
     val locationUrl: String,
-    val repository: RunnerRepository
+    val repository: RunnerRepository,
+    val sshKey: String?
```

Comment:
This should be part of `RunnerRepository`, not `RunnerConfig`.

At the moment it doesn't make a difference because we only ever have at most one Git repository, but eventually we would have multiple in which case it we will want to have one key per repo.